erpnext/stock/doctype/packed_item/packed_item.json

```diff
@@ -215,13 +216,20 @@
    "fieldname": "conversion_factor",
    "fieldtype": "Float",
    "label": "Conversion Factor"
+  },
+  {
+   "fetch_from": "item_code.valuation_rate",
+   "fieldname": "rate",
+   "fieldtype": "Currency",
+   "in_list_view": 1,
+   "label": "Rate"
```

Comment:
```suggestion
   "label": "Rate",
   "read_only": 1
```

Reply:
Is this necessary, considering we're already using Property Setters to achieve this?

Reply:
Initially, the rate column will be editable and the check would be disabled which shouldn't be the case